### ISSUE TITLE
[ SUPPORT REQUEST ] - 

### ISSUE THREAD

**Issue by infonic4321** [Just-Natsuki-Team/NatsukiModDev#798]
# Overview:
this is just a question. I just got Natsuki on to the happy mood today, how much will it take for Natsuki to get to affectionate? i visit her daily and spend like, 2 hours to 4 hours with her.

## Additional information:
[any other information you think is relevant - I.E operating system, date and so on]

**Comment by Blizzardsev** [comment]
Hi @infonic4321 - going forward, please add a title when opening an issue so we can easily identify what the purpose of an issue is without having to navigate into it.

JN is designed to be played over a span of several months, with content being unlocked based on how close the player is to Natsuki, total time spent together, and other criteria - the rate of progression depends on what what actions are taken to increase closeness alongside the number of visits in a given time frame, and is therefore subjective.

In short, the length of each visit doesn't matter, only what you accomplish during each visit. A full list of ways to get closer to Natsuki can be found on the [wiki](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/05:-All-features-(Spoilers)#overview).
